projects/collection-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -360,6 +360,7 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
       }
     }
     this.editorConfig.config = _.assign(this.editorConfig.config, hierarchyConfig);
+    this.editorService.collectionTreeNodes=this.collectionTreeNodes;
```

Comment:
why do we need this? we have `this.nodes.data` inside fancytree component right?

Reply:
remove the `collectionTreeNodes` from editorService.